BrandonBeltran/hackerrank/03/JavaHashset.java

```diff
@@ -0,0 +1,23 @@
+package bootcamp;
+
+import java.util.HashSet;
+import java.util.Scanner;
+
+/* This solution is the one I came on my own without Hackerrank's code.
+*  To be honest, I like it better, I include it for the sake of completion*/
+
+public class JavaHashset {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        int sets = Integer.parseInt(scanner.nextLine());
+
+        HashSet<String> names = new HashSet();
```

Comment:
Code for Interface, not for Implementation
https://www.javaguides.net/2018/06/java-collection-framework-best-practices.html